[2024-12-07T03:37:33.650+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-07T03:37:33.754+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: job_listings_etl_v2.load_data__1 manual__2024-12-07T03:32:36.893822+00:00 [queued]>
[2024-12-07T03:37:33.789+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: job_listings_etl_v2.load_data__1 manual__2024-12-07T03:32:36.893822+00:00 [queued]>
[2024-12-07T03:37:33.793+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2024-12-07T03:37:33.839+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): load_data__1> on 2024-12-07 03:32:36.893822+00:00
[2024-12-07T03:37:33.876+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=9130) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-07T03:37:33.889+0000] {standard_task_runner.py:63} INFO - Started process 9132 to run task
[2024-12-07T03:37:33.889+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'job_listings_etl_v2', 'load_data__1', 'manual__2024-12-07T03:32:36.893822+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/ETL_FINAL_PROJECT (1).py', '--cfg-path', '/tmp/tmpb8vsa749']
[2024-12-07T03:37:33.900+0000] {standard_task_runner.py:91} INFO - Job 56: Subtask load_data__1
[2024-12-07T03:37:34.127+0000] {task_command.py:426} INFO - Running <TaskInstance: job_listings_etl_v2.load_data__1 manual__2024-12-07T03:32:36.893822+00:00 [running]> on host 1ed1f547130e
[2024-12-07T03:37:34.481+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='job_listings_etl_v2' AIRFLOW_CTX_TASK_ID='load_data__1' AIRFLOW_CTX_EXECUTION_DATE='2024-12-07T03:32:36.893822+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-07T03:32:36.893822+00:00'
[2024-12-07T03:37:34.491+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-07T03:37:34.494+0000] {ETL_FINAL_PROJECT (1).py:111} WARNING - No records to load into DEV.RAW_DATA.HISTORICAL_DATA.
[2024-12-07T03:37:34.497+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-12-07T03:37:34.499+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-07T03:37:34.517+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=job_listings_etl_v2, task_id=load_data__1, run_id=manual__2024-12-07T03:32:36.893822+00:00, execution_date=20241207T033236, start_date=20241207T033733, end_date=20241207T033734
[2024-12-07T03:37:34.635+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-07T03:37:34.831+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-07T03:37:34.866+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
